src/common/utils/currencyConvert.ts

```diff
@@ -3,25 +3,18 @@
  */
 import type { CURRENCIES } from "../constants";
 
-const numSatsInBtc = 100_000_000;
-
-const bitcoinToFiat = async (amountInBtc: number | string, rate: number) => {
-  return Number(amountInBtc) * Number(rate);
-};
-
-const satoshisToBitcoin = (amountInSatoshis: number | string) => {
-  return Number(amountInSatoshis) / numSatsInBtc;
+const satsToFiat = async (amount: number | string, rate: number) => {
+  return Number(amount) * rate;
 };
 
 const satoshisToFiat = async ({
-  amountInSats,
+  amount,
```

Comment:
I kinda like names that describe what is in there.

Reply:
I thought we do not need it anymore, because we got rid of the "btc value". But I can keep `amountInSats`

Reply:
I don't care, was just a nit...if one reads it in a year one might not know it. - even though the function is called satoshisInFiat.